File: functions/functions2/ex03.py
"""Write a function to multiplu all numbers of a list"""

def multiply(number_list):
    result = 1
    for i in number_list:
        result *= i
    return result
#----------------------------------

"""------------------------------------
Put the numbers you want to multiply here
--------------------------------------"""
numbers = [3,4,5,6]

print(multiply(numbers))

File: functions/functions2/ex03.c
/* Write a function to multiply all the numbers in a list. */

#include <stdio.h>

/* Multiply the elements of a list*/
int multiply(int *list, int elements);

int main()
{
/*---------------------------------------------------
Put the numbers you want to multiply inside the {}
---------------------------------------------------*/
    int list[] = {3,4,1,3};
   
    printf("Multilpliaction is is: %i\n",multiply(list,sizeof(list)/4));

    return 0;
}

int multiply(int *list, int elements)
{
    int result = list[0];

    for (int i = 1; i < elements; i++)
    {
        result = result * list[i];
    }
    return result;
}